script/prep-travis-release.sh

```diff
@@ -78,7 +78,14 @@ fi
 # Set up the build and deploy layout
 ################################################################################
 
-BASEDIR="${os}/${arch}"
+# Get the version from travis, or default to 0.0.0
+if [ -z $TRAVIS_TAG ]; then
+  VERSION=$TRAVIS_TAG
+else
+  VERSION="v0.0.0"
+fi
```

Comment:
What I would do is allow for linking up with the most recent tag, with a reasonable fallback (for no tags):

```bash
# Allow failure for a moment (for git describe)
set +e
if [ -z "$TRAVIS_TAG" ]; then
    # Version will be the most recent tag, appended with -dev (e.g. 1.0.0-dev)
    OLD_TAG=$(git describe --tags 2> /dev/null)
    VERSION="$OLD_TAG-dev"
    if [ $? != 0 ]; then
        VERSION="dev"
    fi
else
    # We have ourselves a *real* release
    VERSION=$TRAVIS_TAG
fi
set -e
```